src/remix/sdk-setup.ts

```diff
@@ -376,8 +353,32 @@ export async function initializeSentryOnEntryClient(
   );
 }
 
-export async function initializeSentryOnEntryServer(
-  dsn: string,
+export async function updateStartScript(instrumentationFile: string) {
+  const packageJson = await getPackageDotJson();
+
+  if (!packageJson.scripts || !packageJson.scripts.start) {
+    throw new Error(
+      "Couldn't find a `start` script in your package.json. Please add one manually.",
+    );
+  }
+
+  const startCommand = packageJson.scripts.start;
+
+  packageJson.scripts.start = `NODE_OPTIONS='--import ./${instrumentationFile}' ${startCommand}`;
```

Comment:
Should we try to detect for usage of `node xxx` or `remix-serve xxx` before replacing the `start` command?

Reply:
Updated 👍